PersonApi.Tests/AwsIntegrationTests.cs

```diff
@@ -59,7 +62,7 @@ public class TableDef
 
 
     [CollectionDefinition("DynamoDb collection", DisableParallelization = true)]
```

Comment:
Maybe change the collection name to "Aws collection" (and anywhere that uses it)?